templates/cron-prepare.sh.j2

```diff
@@ -33,7 +33,9 @@ function title {
 # popd
 
 title "Postgres DB dump"
-/usr/bin/pg_dump --encoding=UTF8 {{ backups_role_db_name }} > {{ backups_role_tmp_path }}/pg_dump
+for DB in "{{ backups_role_db_names | join('" "')  }}"; do
```

Comment:
Can you document this ´for´ statement please?

Too much `"`... :sweat_smile:

Reply:
As we agreed before, I have explained the hack in a new commit.